src/ontology/fbbt-edit.obo

```diff
@@ -118131,6 +118131,145 @@ relationship: develops_from FBbt:00004945 ! onion stage spermatid
+subset: cur
+is_a: FBbt:00110207 ! enteroendocrine cell of posterior adult midgut epithelium
+intersection_of: FBbt:00110036 ! adult class II enteroendocrine cell
+intersection_of: part_of FBbt:00047155 ! adult posterior midgut
+relationship: expresses http://flybase.org/reports/FBgn0027109 ! NPF
+relationship: expresses http://flybase.org/reports/FBgn0032048 ! Dh31
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-13T13:49:22Z
+
+[Term]
+id: FBbt:00058007
+name: Dh31 cell of the embryonic/larval midgut
+def: "Enteroendocrine cell of the larval midgut that secretes the diuretic hormone Dh31 (FBgn0032048). These cells are particularly abundant in the anterior midgut but are also present in the middle and posterior midgut (Veenstra et al., 2009)." [FlyBase:FBrf0207758]
+subset: cur
+intersection_of: FBbt:00047150 ! larval enteroendocrine cell
+intersection_of: expresses http://flybase.org/reports/FBgn0032048 ! Dh31
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-14T08:32:51Z
+
+[Term]
+id: FBbt:00058008
+name: larval anterior middle midgut junction
+def: "A small region of the larval midgut between the anterior midgut and the middle midgut. The region forms a characteristic U-shaped bend in the larval gut. It contains enteroendocrine cells with a distinct hormonal expression profile from the enteroendocrine cells of the surrounding regions (Lajeunesse et al., 2010; Veenstra et al., 2014)." [FlyBase:FBrf0226048] {def="FlyBase:FBrf0211737"}
+subset: cur
+synonym: "larval midgut junction region" EXACT []
+is_a: FBbt:00001882 ! embryonic/larval midgut
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-14T09:38:55Z
+
+[Term]
+id: FBbt:00058009
+name: embryonic/larval copper cell region
+def: "The most anterior part of the larval middle midgut. That region is composed of cup-shaped, copper-accumulating cells surrounded by interstitial cells. It is highly acidic (Filshie et al., 1971; Dubreuil et al., 1998)." [FlyBase:FBrf0023227, FlyBase:FBrf0100557]
+subset: cur
+is_a: FBbt:00001884 ! embryonic/larval middle midgut
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-14T11:48:57Z
+
+[Term]
+id: FBbt:00058010
+name: embryonic/larval large flat cell region
+def: "A region of the larval middle midgut, located between the copper cell region and the iron cell region. It is composed of large, flat cells (Pouslon and Waterhouse, 1960)." [FlyBase:FBrf0013091]
+is_a: FBbt:00001884 ! embryonic/larval middle midgut
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-14T12:44:32Z
+subsetdef: cur
+
+[Term]
+id: FBbt:00058011
+name: embryonic/larval iron cell region
+def: "The most posterior part of the larval middle midgut. That region is characterized by the presence of iron-accumulating cells (Poulson and Waterhouse, 1960; Missirlis et al., 2007)." [FlyBase:FBrf0013091, FlyBase:FBrf0202180]
```

Comment:
Use this/these, rather than that/those when refering to the class being defined.